site/content/internal/rd-teams.md

```diff
@@ -34,11 +34,12 @@ weight: 110
 ## Cloud Team
 
 * Joram Wilander - Eng Lead
-* Carlos Panato - Engineer/Infrastructure
+* Carlos Panato - SRE
 * Gabe Jackson - Engineer
-* Stylianos Rigas - Engineer/Infrastructure
+* Stylianos Rigas - SRE
 * Ian Whitlock - Engineer
 * Gabriel Sagula - Engineer
+* Angelos Kyratzakos - SRE
```

Comment:
Not sure if he needs to be added here too or not https://developers.mattermost.com/contribute/getting-started/core-committers/#core-committers.